docs: Add a GitHub Issue Explaining the Synthetics Legacy Runtime EOL (#2670)
newrelic/terraform-provider-newrelic: website/synthetics-legacy-runtime-eol.md

Hello!

> As already communicated by New Relic, support for legacy Synthetics runtimes **will reach end-of-life (EOL) on October 22, 2024**. In addition, creating **new** monitors using the legacy runtime **will no longer be supported after June 30, 2024**. This affects Synthetic Monitors running on the legacy runtime, created using any of the following resources:

Author: ```suggestion
As already [communicated by New Relic](https://docs.newrelic.com/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm/#:~:text=On%20October%2022%2C%202024%2C%20New,version%2010%20(or%20older)%20runtimes), support for legacy Synthetics runtimes **will reach end-of-life (EOL) on October 22, 2024**. In addition, creating **new** monitors using the legacy runtime **will no longer be supported after June 30, 2024**. This affects Synthetic Monitors running on the legacy runtime, created using any of the following resources:
```

Seemant's suggestion: add a link to the "already done" communication

> As already communicated by New Relic, support for legacy Synthetics runtimes **will reach end-of-life (EOL) on October 22, 2024**. In addition, creating **new** monitors using the legacy runtime **will no longer be supported after June 30, 2024**. This affects Synthetic Monitors running on the legacy runtime, created using any of the following resources used to manage Synthetic Monitors:

Author: ```suggestion
As already communicated by New Relic, support for legacy Synthetics runtimes **will reach end-of-life (EOL) on October 22, 2024**. In addition, creating **new** monitors using the legacy runtime **will no longer be supported after June 30, 2024**. This affects Synthetic Monitors running on the legacy runtime, created using any of the following resources:
```

suggestion from Seemant

> As already communicated by New Relic, support for legacy Synthetics runtimes **will reach its end-of-life (EOL) on October 22, 2024**. In addition, creating **new** monitors using the legacy runtime **will no longer be supported after June 30, 2024**. This would affect Synthetic Monitors running on the legacy runtime, created using any of the following resources used to manage Synthetic Monitors:

Author: ```suggestion
As already communicated by New Relic, support for legacy Synthetics runtimes **will reach end-of-life (EOL) on October 22, 2024**. In addition, creating **new** monitors using the legacy runtime **will no longer be supported after June 30, 2024**. This affects Synthetic Monitors running on the legacy runtime, created using any of the following resources used to manage Synthetic Monitors:
```

(corrections suggested by @seemantk)

- [newrelic_synthetics_monitor](https://registry.terraform.io/providers/newrelic/newrelic/3.36.1/docs/resources/synthetics_monitor)
- [newrelic_synthetics_script_monitor](https://registry.terraform.io/providers/newrelic/newrelic/3.36.1/docs/resources/synthetics_script_monitor)
- [newrelic_synthetics_step_monitor](https://registry.terraform.io/providers/newrelic/newrelic/3.36.1/docs/resources/setics_step_monitor)
- [newrelic_synthetics_cert_check_monitor](https://registry.terraform.io/providers/newrelic/newrelic/3.36.1/docs/resources/synthetics_cert_check_monitor)
- [newrelic_synthetics_broken_links_monitor](https://registry.terraform.io/providers/newrelic/newrelic/3.36.1/docs/resources/synthetics_broken_links_monitor)

> In light of the above, kindly **upgrade your Synthetic Monitors to the new runtime** at the earliest (before the date of the EOL), if they are still using the legacy runtime. Check out [this page](https://forum.newrelic.com/s/hubtopic/aAXPh0000001brxOAA/upcoming-endoflife-legacy-synthetics-runtimes-and-cpm) for more details on the EOL, action needed (specific to monitors using public and private locations), relevant resources, and more.

Author: ```suggestion
In light of the above, kindly **upgrade your Synthetic Monitors to the new runtime** before the EOL date. Check out [this page](https://forum.newrelic.com/s/hubtopic/aAXPh0000001brxOAA/upcoming-endoflife-legacy-synthetics-runtimes-and-cpm) for more details on the EOL, actions needed, relevant resources, and more.
```

(corrections suggested by @seemantk)

We will synchronize relevant changes in the New Relic Terraform Provider with the Synthetics team. Watch this space for more updates.

> The article linked above comprises key details around the EOL and has been published by New Relic Synthetics. We shall also continue to share any useful information around the EOL via this thread, or via the documentation of the New Relic Terraform Provider.

Author: ```suggestion
We shall also continue to share any useful information around the EOL via this thread, or via the documentation of the New Relic Terraform Provider.
```

Seemant's suggestion

> The article linked above comprises key details around the EOL and has been published by New Relic Synthetics. We shall also continue to share any useful information around the EOL via this thread, or via the documentation of the New Relic Terraform Provider, if we find such information can be useful to a larger audience.

Author: ```suggestion
The article linked above comprises key details around the EOL and has been published by New Relic Synthetics. We shall also continue to share any useful information around the EOL via this thread, or via the documentation of the New Relic Terraform Provider.
```

(correction suggested by @seemantk)

Hello!

> 

Author: ```suggestion
```

Hello!

> This issue/post has been created to convey details about the upcoming end-of-life (EOL) of the Synthetics Legacy Runtime, which would, post the end-of-life, affect all Synthetic Monitors created using the legacy runtime.

Author: ```suggestion
```
